carbon_black_cloud/icon_carbon_black_cloud/util/agent_typer.py

```diff
@@ -0,0 +1,26 @@
+from ipaddress import ip_address
+import re
+
+MAC_ADDRESS = "Mac Address"
+IP_ADDRESS = "IP Address"
+HOSTNAME = "Host Name"
+DEVICE_ID = "Device ID"
```

Comment:
This could be a great area to utilize an enum

Reply:
I like the way this is called over the enum. 

agent_typer.<type> 

vs 

agent_typer.<enum>.<type>